app/medInria/medMainWindow.cpp

```diff
@@ -690,6 +690,10 @@ void medMainWindow::closeEvent(QCloseEvent *event)
         return;
     }
     this->saveSettings();
+
+    dtkInfo() << "### Application is closing...";
+    dtkInfo() << "####################################";
+
```

Comment:
for some reason it doesn't work, but I'm working on something that might fix that